assignments/4. operators/loops.md

```diff
@@ -14,16 +14,48 @@ const ACCESSORY_PRICE = 9.99;
 var bank_balance = 303.91;
 var amount = 0;
 // your code goes here
+for (; PHONE_PRICE <= bank_balance; bank_balance - PHONE_PRICE) {
+    bank_balance = bank_balance - PHONE_PRICE;
+    amount = amount + PHONE_PRICE;
+
+    for (; ACCESSORY_PRICE <= SPENDING_THRESHOLD; bank_balance - ACCESSORY_PRICE) {
+        bank_balance = bank_balance - ACCESSORY_PRICE;
+        amount = amount + ACCESSORY_PRICE;
+        break;
+    }
+}
+// Calculate tax function
+let total = 0;
+let calculateTax = (amount) => {
+    total = amount * TAX_RATE;
+    return total;
+}
+
+// Formate amount function
+let formatAmount = (amount) => `$${amount.toFixed(2)}`;
+
+// Check amount against account bal
+let permission = (total <= bank_balance) ? 'Yes, you can buy it' : 'You don\'t have enough money in your account'; 
```

Comment:
Do it again going through the requirements. The answer should be `$334.76`. Using your program I am getting `219.95999999999998`. The first thing is ` You will keep purchasing phones until you run out of bank balance.` the bank balance is `303.91`. So how can it stop before that. Only after you go beyond the bank balance it should stop you.